lib/main.dart

```diff
@@ -12,8 +15,9 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Kody Testowe Poli',
-      home: BarcodesPage(barcodes: BarcodeItem.barcodes),
+      title: (Translations.of(context).appTitle),
```

Comment:
Zamiast powtarzać `Translations.of(context)` wynieś to do zmiennej na początku metody.
`Translations translations = Translations.of(context)` i wtedy będzie mógł używać tego tak `translations.appTitle`.